source/clear-linux/tutorials/multi-boot/multi-boot-ubuntu.rst

```diff
@@ -56,87 +56,84 @@ tutorial.
+   Ubuntu as a boot option:
 
    #. Boot into Ubuntu.
 
    #. Log in.
 
-   #. Locate the Ubuntu :file:`grub.cfg` file in the :file:`/boot/grub/`
-      directory and look for the :guilabel:`menuentry` section. In Figure 5, the
-      highlighted lines identify the kernel, the :file:`initrd` files, the
-      root partition UUID, and the additional parameters used. Use this
-      information to create a new Systemd-Boot entry for Ubuntu.
-
-      .. figure:: figures/multi-boot-ubuntu-5.png
+#. Open a Terminal, and cd into the root directory:
 
-         Figure 5: Ubuntu: grub.cfg file.
-
-   #. Copy the kernel and the :file:`initrd` file to the EFI partition.
+   .. code-block:: bash
 
-      .. code-block:: bash
+      cd  /
 
-         sudo cp /boot/vmlinuz-4.8.0-36-generic.efi.signed /boot/efi
+#. Next, create a boot entry for Ubuntu to invoke grub, using this format:
 
-         sudo cp /boot/initrd.img-4.8.0-36-generic /boot/efi
+   +---------+------------------------------------+
+   | Setting | Description                        |
+   +=========+====================================+
+   | title   | Text to show in the boot menu      |
+   +---------+------------------------------------+
+   | efi     | Linux bootloader                   |
+   +---------+------------------------------------+
 
-   #. Create a boot entry for Ubuntu. At a minimum, the file must contain
-      these settings:
+   .. note::
 
-      +---------+------------------------------------+
-      | Setting | Description                        |
-      +=========+====================================+
-      | title   | Text to show in the boot menu      |
-      +---------+------------------------------------+
-      | linux   | Linux kernel image                 |
-      +---------+------------------------------------+
-      | initrd  | initramfs image                    |
-      +---------+------------------------------------+
-      | options | Options to pass to the EFI program |
-      |         | or kernel boot parameters          |
-      +---------+------------------------------------+
+      See the `systemd boot loader documentation`_ for additional details.
 
-      See the `systemd boot loader documentation`_ for additional
-      details.
+#. To do so, enter the command:
 
-      The *options* parameters must specify the root partition UUID and
-      any additional parameters that Ubuntu requires.
+   .. code-block:: bash
 
-      .. note:: The root partition UUID used below is unique to this example.
+      sudoedit /boot/efi/loader/entries/ubuntu.conf
 
-      .. code-block:: bash
+#. Add the following lines to the :file:`ubuntu.conf` file:
 
-         sudoedit /boot/efi/loader/entries/ubuntu.conf
+   .. code-block:: bash
 
-      Add the following lines to the :file:`ubuntu.conf` file:
+      title Ubuntu 18.04 LTS Desktop
 
-      .. code-block:: console
+      efi /EFI/ubuntu/grubx64.efi
```

Comment:
@bwarden, @evil-brain, Can either of you see why the above command won't work--or requires an additional step--after installing more OSes after Clear Linux OS? This Ubuntu multi-boot was confirmed to work prior to release of new `live-desktop` and `live-server` isos; however, it's not working now. 
Full render, see: https://github.com/mvincerx/clear-linux-documentation/blob/8019b1a16f74c2cad092ab85bd11fdea0a89a628/source/clear-linux/tutorials/multi-boot/multi-boot-ubuntu.rst